src/main/java/org/opensearch/timeseries/constant/CommonName.java

```diff
@@ -33,6 +33,9 @@ public class CommonName {
     // ======================================
     // Index name
     // ======================================
+
+    public static final String ALL_AD_DETECTOR_INDEX_PATTERN = ".opendistro-anomaly-detectors*";
```

Comment:
We don't need ALL_AD_DETECTOR_INDEX_PATTERN.

Reply:
Removed